dist/assets/examples/en/03_Arrays/02_Array_Objects.js

```diff
@@ -31,15 +31,17 @@ function draw() {
 }
 
 
-function Module(_xOff, _yOff, _x, _y, _speed, _unit) {
-  this.xOff = _xOff;
-  this.yOff = _yOff;
-  this.x = _x;
-  this.y = _y;
-  this.speed = _speed;
-  this.unit = _unit;
-  this.xDir = 1;
-  this.yDir = 1;
+class Module {
+  constructor(_xOff, _yOff, _x, _y, _speed, _unit) {
```

Comment:
```suggestion
  constructor(xOff, yOff, x, y, speed, unit) {
```

Not sure if this is silly, but I find that in JS it's a little friendlier to just use the variable name without the underscore. These examples came from Processing where the distinction is necessary but here since `this.` is required it no longer is!